src/util/select-tools.ts

```diff
@@ -0,0 +1,28 @@
+import { MultiDirectedGraph } from 'graphology';
+import { NodeAttributes, EdgeAttributes, GraphAttributes } from '../constants/constants';
+
+const inRange = (x1: number, y1: number, x2: number, y2: number, xq: number, yq: number) => {
+    return x1 <= xq && xq <= x2 && y1 <= yq && yq <= y2;
+};
+
+export const HandleSelectTool = (
```

Comment:
`src/util/graph.ts` might be a better place :)

Reply:
Shall I move all these to graph.tsx?

Reply:
Yeah, graph.ts should be an all in one place for all graph-related helper functions :)

Reply:
Also, the function name should be `handleSelectTool` :)

Reply:
Done.